woocommerce/woocommerce-android
# Improve order status update snackbars

Closes: #4249 

### Changes

#### Functional

`🎉 Order completed!` snackbar won't be displayed after a successful update on the backend but after marking the order as complete from the fulfilling screen.

#### UI-only

Order status update from dialog, `Order status updated` will be displayed instead of `Order status changed to {status}`

### How to test

Please check the below demo and check if the scenarios presented in "after" column work as intended.

### Demo

| | Before | After |
| --- | --- | --- |
| Change status to other than `Completed` | ![before_normal_change](https://user-images.githubusercontent.com/5845095/123842727-6493a080-d911-11eb-8a50-13df96fc93e6.gif) | ![after_normal_change](https://user-images.githubusercontent.com/5845095/123842767-6f4e3580-d911-11eb-85a3-0cc916c14703.gif) |
| Change status to `Completed` from dialog | ![before_dialog_completed](https://user-images.githubusercontent.com/5845095/123842844-85f48c80-d911-11eb-9f1b-3d0c98ecd21a.gif) | ![after_dialog_completed](https://user-images.githubusercontent.com/5845095/123842887-8d1b9a80-d911-11eb-8c41-119a046a3e29.gif) |
| Change status to `Completed` from fulfill screen | ![before_fulfill_completed](https://user-images.githubusercontent.com/5845095/123842915-9573d580-d911-11eb-9578-6c869e84dc8a.gif) | ![after_fulfill_completed](https://user-images.githubusercontent.com/5845095/123842942-9ad12000-d911-11eb-9c6c-b83d2dd45aa9.gif) |



Update release notes:
